internals/maintenance_scripts.py

```diff
@@ -803,6 +804,41 @@ def calc_needs_work_started_on(
                if v.state == Vote.NEEDS_WORK)
 
 
+class FetchWebdxFeatureId(FlaskHandler):
+
+  def get_template_data(self, **kwargs) -> str:
+    """Fetch the complete list of Webdx feature ID available from
+    webstatus.dev APIs and store them in datastore.
+    """
+    self.require_cron_header()
+
+    if settings.UNIT_TEST_MODE:
+      webstatus_dev_url = 'https://api.server.test'
+    else:
+      webstatus_dev_url = 'https://api.webstatus.dev'
+    client = DefaultApi(ApiClient(Configuration(host=webstatus_dev_url)))
+
+    all_data_list: list[Feature] = []
+    page_token: str | None = None
+    is_first: bool = True
+    while is_first or page_token:
+        try:
+            resp = client.list_features(page_token=page_token, page_size=100)
+            all_data_list.extend(resp.data)
+            page_token = resp.metadata.next_page_token
+            is_first = False
+        except ApiException as e:
+          logging.error(
+            'Could not fetch from %s?page_token=%s: %s', webstatus_dev_url, page_token, e
+          )
+          return 'Running FetchWebdxFeatureId() job failed.'
+
+    # TODO(kyleju): save it to datastore.
+    feature_ids_list = [feature_data.feature_id for feature_data in all_data_list]
```

Comment:
Note to self: While webstatus.dev supports sorting by name, it does not support sorting by id. Filed https://github.com/GoogleChrome/webstatus.dev/issues/1087